serde_derive/src/internals/attr.rs

```diff
@@ -777,7 +789,6 @@ pub enum Default {
 }
 
 impl Default {
-    #[cfg(feature = "deserialize_in_place")]
```

Comment:
Is this change intended?

Reply:
I believe so, deciding whether a field during `derive(Deserialize)` is transparent now checks `field.attrs.default().is_none()`.